rakudup

```diff
@@ -24,51 +23,51 @@ if [[ ! $(perl -v) =~ "Perl 5" ]]; then
     exit 1
 fi
 
-TMP=${RAKUDO_ROOT}/tmp
+TMP=$RAKUDO_ROOT/tmp
 
-RELEASE=$(git clone -q --no-checkout --depth 1 git@github.com:rakudo/rakudo.git ${TMP} > /dev/null 2>&1 && cd $dir && git -C ${TMP} show master:VERSION && rm -rf ${TMP})
+RELEASE=$(git clone -q --no-checkout --depth 1 git@github.com:rakudo/rakudo.git -- "$TMP" > /dev/null 2>&1 && cd -- "$dir" && git -C "$TMP" show master:VERSION && rm -rf -- "$TMP")
 
-RAKUDO_SRC="${RAKUDO_ROOT}/${RELEASE}/src"
-RAKUDO_INSTALL="${RAKUDO_ROOT}/${RELEASE}/install"
+RAKUDO_SRC="$RAKUDO_ROOT/$RELEASE/src"
+RAKUDO_INSTALL="$RAKUDO_ROOT/$RELEASE/install"
 
-if [[ -d ${RAKUDO_INSTALL} ]]; then
+if [[ -d $RAKUDO_INSTALL ]]; then
     echo "${RELEASE} already installed not updating"
     exit 1
 fi
 
-echo "Getting Rakudo ${RELEASE}"
+echo "Getting Rakudo $RELEASE"
 
 git clone -q -c advice.detachedHead=false --depth=1 -b ${RELEASE} https://github.com/rakudo/rakudo.git ${RAKUDO_SRC} 
 
-cd ${RAKUDO_SRC}
-echo "Building Rakudo ${RELEASE}. This will take several minutes on a modern CPU."
-echo "'tail -f ${RAKUDO_SRC}/build.log' for progress"
+cd -- "$RAKUDO_SRC" || exit 1
```

Comment:
I'll add the exit but don't care about the case where directory starts with dash! (Although it is interesting!)

Reply:
What? But that's the whole point…